### ISSUE TITLE
mesos-hdfs keeps giving me error when I start

### ISSUE THREAD

**Issue by mesosapprentice** [mesosphere/hdfs#189]
I keep getting the following error when I attempt to start mesos-hdfs.. 

Error: Could not find or load main class org.apache.mesos.hdfs.scheduler.Main

I am guessing its the java app not being able to find the right library, but aren't all the libraries packaged in the tarball?

**Comment by elingg** [comment]
hm, yes it should be built with the package. Could you try with the latest version of the code? Are you running on master? Were you able to build successfully?

**Comment by mesosapprentice** [comment]
So I went ahead and downloaded your code again. I am now getting this error.. I was running it last week with the nocomplie option since gradle was giving me fits. I just ran the build and I am getting this. Am I missing any prerequisite software? I am new to mesos, so I am definitely thinking I am making a mistake.. this is on a VM by the way. 

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':hdfs-commons:compileJava'.
> Could not find tools.jar

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

**Comment by mesosapprentice** [comment]
Okay so I am making progress here.. I removed and reinstalled java to my mesos-master. CentOS had installed OpenJDK 7 by default. I downloaded JDK 8 from Oracle and installed that. Once I did that, I was able to do the build with gradle and then I could launch. Now I just need to tweak my site files so that it work with my configuration that I am setting up. Thanks for getting back to me. As I figured it was a problem on my end, but if I can provide something here to help others, then I guess I at least accomplished that. Guess I just needed a sounding board to bounce the idea around..
